packages/fs.go

```diff
@@ -35,23 +36,29 @@ func NewExtractedPackageFileSystem(p *Package) (*ExtractedPackageFileSystem, err
 }
 
 func (fs *ExtractedPackageFileSystem) Stat(name string) (os.FileInfo, error) {
-	return os.Stat(filepath.Join(fs.path, name))
+	return os.Stat(path.Join(fs.path, name))
 }
 
 func (fs *ExtractedPackageFileSystem) Open(name string) (PackageFile, error) {
-	return os.Open(filepath.Join(fs.path, name))
+	return os.Open(path.Join(fs.path, name))
 }
 
 func (fs *ExtractedPackageFileSystem) Glob(pattern string) (matches []string, err error) {
-	matches, err = filepath.Glob(filepath.Join(fs.path, pattern))
+	matches, err = filepath.Glob(path.Join(fs.path, pattern))
 	if err != nil {
 		return
 	}
 	for i := range matches {
-		match, err := filepath.Rel(fs.path, matches[i])
```

Comment:
If we keep using `filepath` here, we would be able to continue using `filepath.Rel`.